circus-train-core/pom.xml

```diff
@@ -141,7 +141,7 @@
     <!-- SSH library -->
```

Comment:
Comment here is out of date, I'd suggest removing it to prevent this happening again in future, it's clear from the artifact ID what it is.